infrastructure/template.yaml

```diff
@@ -384,115 +314,111 @@ Resources:
     Properties:
       Description: Role to allow API gateway to execute step function
       AssumeRolePolicyDocument:
-        Version: "2012-10-17"
+        Version: 2012-10-17
         Statement:
           - Effect: Allow
+            Action: sts:AssumeRole
             Principal:
-              Service:
-                - apigateway.amazonaws.com
-            Action:
-              - "sts:AssumeRole"
+              Service: apigateway.amazonaws.com
       Policies:
         - PolicyName: AllowStateMachineInvoke
           PolicyDocument:
-            Version: "2012-10-17"
+            Version: 2012-10-17
             Statement:
               - Effect: Allow
-                Action:
-                  - "states:StartExecution"
-                  - "states:StartSyncExecution"
                 Resource: "*"
-      PermissionsBoundary:
-        !If [
-          UsePermissionsBoundary,
-          !Ref PermissionsBoundary,
-          !Ref AWS::NoValue,
-        ]
+                Action:
+                  - states:StartExecution
+                  - states:StartSyncExecution
+      PermissionsBoundary: !If
+        - UsePermissionsBoundary
+        - !Ref PermissionsBoundary
+        - !Ref AWS::NoValue
 
   NinoAttemptsTable:
-    Type: "AWS::DynamoDB::Table"
+    Type: AWS::DynamoDB::Table
     Properties:
-      TableName: !Sub "${AWS::StackName}-nino-attempts"
-      BillingMode: "PAY_PER_REQUEST"
+      TableName: !Sub ${AWS::StackName}-nino-attempts
+      BillingMode: PAY_PER_REQUEST
       AttributeDefinitions:
-        - AttributeName: "id"
-          AttributeType: "S"
+        - AttributeName: id
+          AttributeType: S
       KeySchema:
-        - AttributeName: "id"
-          KeyType: "HASH"
+        - AttributeName: id
+          KeyType: HASH
 
   NinoUsersTable:
-    Type: "AWS::DynamoDB::Table"
+    Type: AWS::DynamoDB::Table
     Properties:
-      TableName: !Sub "${AWS::StackName}-nino-users"
-      BillingMode: "PAY_PER_REQUEST"
+      TableName: !Sub ${AWS::StackName}-nino-users
+      BillingMode: PAY_PER_REQUEST
       AttributeDefinitions:
-        - AttributeName: "sessionId"
-          AttributeType: "S"
+        - AttributeName: sessionId
+          AttributeType: S
       KeySchema:
-        - AttributeName: "sessionId"
-          KeyType: "HASH"
+        - AttributeName: sessionId
+          KeyType: HASH
 
   MatchingFunction:
     Type: AWS::Serverless::Function
-    Properties:
-      Handler: lambdas/matching/src/matching-handler.lambdaHandler
     Metadata:
       BuildMethod: esbuild
       BuildProperties:
-        Minify: true
```

Comment:
Any reason we are removing these from `BuildProperties`? I think minify should be kept, unless it is still doing that?

Reply:
These are the default values so there's no need to specify them.
Full spec of the default values in the AWS documentation:
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-using-build-typescript.html

Now, there is some value in being explicit but the templates currently are too big to be read easily so anything we can do to make them smaller helps to maintain and understand them.